project/.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```

Comment:
excePt?

Reply:
Nope, this is "[exempt](https://github.com/probot/stale/blob/cd1e7623c6971731d4c19846d62b4dd383616171/lib/stale.js#L68)".